connectivity/check/features.go

```diff
@@ -162,6 +163,15 @@ func (ct *ConnectivityTest) extractFeaturesFromConfigMap(ctx context.Context, cl
 		Mode:    mode,
 	}
 
+	mode = "disabled"
```

Comment:
Why is this not capitalized, when all the other ones are `"Disabled"`

Reply:
Good catch, probably it's time for a const.